sway-lib-std/src/lib.sw

```diff
@@ -10,6 +10,7 @@ dep mem;
 dep alloc;
 dep contract_id;
 dep constants;
+dep external;
```

Comment:
What is the reasoning for calling the library `external`?

Reply:
Yeah, see https://github.com/FuelLabs/sway/issues/3081. I couldn't think of a better name. This reads the bytecode root of an "external" contract (although as mentioned in the issue, not strictly so)

I imagine this module eventually containing the wrappers for `CCP` and `CSIZ` , (and _maybe_ `LDC`) . Can bikeshed on the name in the issue.

Reply:
How about `bytecode.sw` or something like that? All of these ops seem to be doing something related bytecode.